docs/examples/EdgeX_ExampleMQTT_configuration.toml

```diff
@@ -4,7 +4,7 @@ Port = 49982
 ConnectRetries = 3
 Labels = []
 OpenMsg = "device mqtt started"
-MaxResultCount = 50000
+ReadMaxLimit = 256
```

Comment:
This needs to be changed back, the new key name is `MaxResultCount`. Also please don't set the example to something low like 256, the default of 50000 is a reasonable default

Reply:
I pulled this from https://github.com/edgexfoundry/device-mqtt-go/blob/master/cmd/res/example/configuration.toml at @cloudxxx8's recommendation. Cloud, can you confirm whether I should make the change that Ian is suggesting?

Reply:
Hmm I suppose this is the configuration.toml for a device service and not for the device services. Perhaps we need to propogate https://github.com/edgexfoundry/edgex-go/issues/1326 to the device service SDKs as well

Reply:
I asked about this on the #device-services channel on slack for clarification

Reply:
I've removed this line as recommended by @cloudxxx8